gremlin-test/src/main/java/org/apache/tinkerpop/gremlin/structure/PropertyTest.java

```diff
@@ -59,14 +59,40 @@
     public static class BasicPropertyTest extends AbstractGremlinTest {
         @Test
         @FeatureRequirementSet(FeatureRequirementSet.Package.VERTICES_ONLY)
-        public void shouldHaveStandardStringRepresentation() {
+        public void shouldHaveStandardStringRepresentationForVertexProperty() {
             final Vertex v = graph.addVertex("name", "marko");
             final Property p = v.property("name");
             assertEquals(StringFactory.propertyString(p), p.toString());
         }
 
         @Test
         @FeatureRequirementSet(FeatureRequirementSet.Package.VERTICES_ONLY)
+        public void shouldHaveTruncatedStringRepresentationForVertexProperty() {
+            final Vertex v = graph.addVertex("name", "maria de la santa cruz rosalina agnelia rodriguez cuellar rene");
+            final Property p = v.property("name");
+            assertEquals(StringFactory.propertyString(p), p.toString());
```

Comment:
The only thing that this test validates is that the toString has the same output as the StringFactory which may be all well and good.  However, if StringFactory changes, the would still pass, but there could be an end-user display impact that isn't caught.  I'm not against it as long as we all understand the implications. I prefer hardcoded expectations to catch these things.